pip/download.py

```diff
@@ -593,6 +556,11 @@ def resp_read(chunk_size):
                     break
                 yield chunk
 
+    def written_chunks(chunks):
```

Comment:
It's named after its return value. I do that a lot so it's clear at the callsite what the value coming back it. http://www.grinchcentral.com/function-names-to-verb-or-not-to-verb. Feel strongly?